### PR TITLE
Continuous Deployment

### PR DESCRIPTION
**Pull request by jswny**
Adds automatic deployment for the bot. Uses the production token to deploy from develop automatically.

### TODO before merging
- [x] Make sure the deploy actually works
- [x] Update deploy condition to deploy on develop (testing with the beta bot token currently)
- [x] Update bot token to production token

### CODE REVIEW

**Comment by ironchefpython** [comment]
No.  This needs to be ansible and a bunch of other stuff.

**Comment by jswny** [comment]
Waiting on @samcat116 to give me a VM that isn't ARM. Other than that this should be good to go after that is done (just need to edit some temporary values).

**Comment by jswny** [comment]
> No. This needs to be ansible and a bunch of other stuff.

Sam has my permission to nuke this as soon as he's done with his stuff 😎

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -124,10 +126,50 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.CR_PAT }}
       - name: Build and Push to GitHub Container Registry
+        id: docker-build
         uses: docker/build-push-action@v2
         with:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.image-metadata.outputs.tags }}
           labels: ${{ steps.image-metadata.outpus.labels }}
```
Comment (cooperpellaton) [inline .github/workflows/ci.yml:134]: Typo here. `outpus` -> `outputs`.

**Review by cooperpellaton — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jswny) [reply .github/workflows/ci.yml]: Fixed, good catch!

**Review by ironchefpython — APPROVED** [review]
This is all stupid and pointless when you can just use python to run it.

Approved.

**Comment by jswny** [comment]
> This is all stupid and pointless when you can just use python to run it.
> 
> Approved.

I'll believe it when I see it ;)

**Comment by ironchefpython** [comment]
`/usr/bin/python3.9 -m poetry run python -m nachomemes.bot`

Checkmate Atheist

**Comment by jswny** [comment]
> `/usr/bin/python3.9 -m poetry run python -m nachomemes.bot`
> 
> Checkmate Atheist

This is a human rights violation

**Comment by ironchefpython** [comment]
\*beep-boop\* 

The robot overlords care not for your rights.

**Pull request merged**